flybywiresim/a32nx
# build: bump @flybywiresim/eslint-config to 0.2.2

## Summary of Changes


* Waiting for #5541
* Bump `@flybywiresim/eslint-config` to 0.2.2.

Discord username (if different from GitHub): someperson#4953

## Testing Instructions

- Make sure **all** flyPad features still work
- Make sure DCDU still has same very basic functionality
- Make sure SD pages still work


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
